pkg/cmd/roachtest/testdata/regression.diffs

```diff
@@ -226152,30 +226084,112 @@ diff -U3 --label=/mnt/data1/postgres/src/test/regress/expected/rangefuncs.out --
  drop function rngfuncbar();
  -- check handling of a SQL function with multiple OUT params (bug #5777)
  create or replace function rngfuncbar(out integer, out numeric) as
- $$ select (1, 2.1) $$ language sql;
- select * from rngfuncbar();
-- column1 | column2 
-----------+---------
--       1 |     2.1
--(1 row)
--
-+ERROR:  function rngfuncbar() does not exist
+@@ -2344,115 +1929,182 @@
  create or replace function rngfuncbar(out integer, out numeric) as
  $$ select (1, 2) $$ language sql;
  select * from rngfuncbar();  -- fail
 -ERROR:  function return row and query-specified return row do not match
 -DETAIL:  Returned type integer at ordinal position 2, but query expects numeric.
-+ERROR:  function rngfuncbar() does not exist
++ERROR:  internal error: invalid datum type given: INT8, expected DECIMAL
```

Comment:
I'll open an issue for this.

Reply:
I think it's the same as #113186 and #114846.

Reply:
Great, thanks.